addons/website_sale/static/src/js/website_sale.js

```diff
@@ -169,11 +169,27 @@ publicWidget.registry.WebsiteSale = publicWidget.Widget.extend(VariantMixin, {
     start: function () {
         var def = this._super.apply(this, arguments);
 
+        var hash = window.location.hash.substring(1);
+        if (hash) {
```

Comment:
What if an user adds an anchor in the description of its product and links to it? Maybe change to `#attributes=x,y,z` (or a better name if you have one) and parse the hash?